test-engineering/contract-tests/kinto-setup/main.py

```diff
@@ -35,6 +35,9 @@ def main(
                 mimetype="application/json",
                 filecontent=data_file.read_bytes(),
             ),
+            data_type="offline-expansion-data"
+            if "offline" in data_file.stem
+            else "data",
```

Comment:
I had a thought that an assignment with a common function, that assumes file name convention, but **removes the custom condition** would be more ideal for test. It also allows for more data types should the need arise.

Something like:

```suggestion
            data_type=data_file.stem[:-3],
```
```suggestion
            data_type=data_file.stem.rsplit('-', 1)[0],
```
```suggestion
            # Needs 'import re'
            data_type=re.match("^(?P<data_type>.*)-\d\d$", data_file.stem).group("data_type"),
```